crates/bevy_render/src/renderer/render_device.rs

```diff
@@ -51,6 +51,19 @@ impl RenderDevice {
         self.device.create_shader_module(desc)
     }
 
+    /// Creates a [`ShaderModule`](wgpu::ShaderModule) from either SPIR-V or WGSL source code without runtime checks.
+    ///
+    /// # Safety
+    ///
```

Comment:
This should fully detail the safety invariant by itself instead of linking to wgpu's docs.